src/App.jsx

```diff
@@ -1,10 +1,71 @@
 import './App.css'
+import pokemonlogo from './assets/International_Pokémon_logo.svg.png'
+import axios from "axios";
+import {useEffect, useState} from "react";
+import Pokemon from "./components/Pokemon.jsx";
+
+
+
 
 function App() {
 
+
+
+    const [pokemon,setPokemon] = useState([])
+    const [endpoint,setEndpoint] = useState("https://pokeapi.co/api/v2/pokemon/")
+
+    useEffect(() => {
+        async function findpokemon(){
+            try{ const response = await axios.get(endpoint)
+                setPokemon(response.data)
+            }catch(e){
```

Comment:
Je kunt met een useState bijhouden of er een error is opgetreden of niet en op basis van die waarde een `{error && <p>Something went wrong.</p>}` of iets dergelijks tonen

Reply:
Ditzelfde geldt ook voor loading. Aan het begin van de `findPokemon` functie zet je `setLoading(true);` en op het eind op false. Op basis van die boolean kun je dan ook weer iets tonen